public/themes/pterodactyl/js/frontend/console.js

```diff
@@ -47,7 +47,7 @@ $(document).ready(function () {
 
     $terminalInput.focus();
     $('.terminal_input--prompt, #terminal_input, #terminal, #terminalNotify').on('click', function () {
```

Comment:
Remove the whole statement instead of commenting it out.

Reply:
@lancepioch see my comment in the actual PR about just removing `#terminal` from the list.

Reply:
Removing #terminal should work.